lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -386,6 +392,13 @@ def main():
                 post_action=lambda h: time.sleep(module.params['poll_interval']),
                 fail_condition=failed_state,
             )
+        elif state == 'iscsidiscover':
+            ret = hosts_module.action(
+                # This is a synchronize action and we only want to
+                # know if we got 200 response code
+                action='iscsi_discover',
+                iscsi=otypes.IscsiDetails(address=module.params['iscsi']['address'])
```

Comment:
Regarding login, if we re-login I think that this should be handled through the engine, what do you think?